test/unit/BCoWPool.t.sol

```diff
@@ -13,11 +16,15 @@ abstract contract BaseCoWPoolTest is BasePoolTest {
   bytes32 public domainSeparator = bytes32(bytes2(0xf00b));
   address public vaultRelayer = makeAddr('vaultRelayer');
 
+  MockBCoWPool bCoWPool;
+
   function setUp() public override {
     super.setUp();
     vm.mockCall(cowSolutionSettler, abi.encodePacked(ISettlement.domainSeparator.selector), abi.encode(domainSeparator));
     vm.mockCall(cowSolutionSettler, abi.encodePacked(ISettlement.vaultRelayer.selector), abi.encode(vaultRelayer));
-    bPool = MockBPool(address(new MockBCoWPool(cowSolutionSettler)));
+    bCoWPool = new MockBCoWPool(cowSolutionSettler);
+    bPool = MockBPool(address(bCoWPool));
```

Comment:
Now that #84 is merged you dont need 2 pools. Because `IBCowPool` inheriths `IBPool`

Reply:
:fire:

Reply:
mhhh there's not much DX improvement in storing it on `bPool` and requiring to `MockBCoWPool(bPool)` to access to those methods, i'm ok with this approach